migrations/migrations.sql

```diff
@@ -3925,6 +3925,11 @@ do $$
       alter table "accounts" add column "isFlagged" boolean;
     end if;
 
+    -- Add "blockedEmails" to systemSettings
+    if not exists (select 1 from information_schema.columns where table_name = 'systemSettings' and column_name = 'blockedEmails') then
+      alter table "systemSettings" add column "blockedEmails" TEXT;
+    end if;
+
```

Comment:
We don't add system settings in migrations.sql, we have a different file to do that i.e `system_settings.sql.template`

Reply:
This might break things if we do it like this.